lib/ibandit/iban_assembler.rb

```diff
@@ -67,7 +63,8 @@ def self.assemble_sm_bban(opts)
     ##################
 
     def self.can_assemble?(local_details)
-      SUPPORTED_COUNTRY_CODES.include?(local_details[:country_code]) &&
+      Constants::SUPPORTED_COUNTRY_CODES.
+        include?(local_details[:country_code]) &&
```

Comment:
This formatting is a bit funky. Maybe extract the first condition into a `supported_country_code?` method?